doc/source/data/dataset-tensor-support.rst

```diff
@@ -439,6 +439,22 @@ If working with in-memory Pandas DataFrames that you want to analyze, manipulate
     print(read_df.equals(df))
     # -> True
 
+.. _disable_tensor_extension_casting:
```

Comment:
FYI @ericl is doing major rewriting in https://github.com/ray-project/ray/pull/26952/files .

Reply:
@bveeramani can you have a followup PR for this file change after https://github.com/ray-project/ray/pull/26952 is merged, or comment there for this change? This can help avoid conflict when @ericl is doing the same change in the file.

Reply:
> comment there for this change

Yeah, I'll do so and also ping Eric directly. I think it's important to include this information because it's not documented anywhere.